src/components/button/button-theme.scss

```diff
@@ -4,7 +4,7 @@
     &:hover {
       background-color: '{{background-500-0.2}}';
     }
-    &.md-focused {
+    &.md-focused:not(.md-no-focus) {
```

Comment:
This line actually shouldn't mind anymore. The scenario itself will be present, if the button is focused and the user did disable the focus.

The Class won't just match with any selectors because of the `:not`